crates/topos-tce-storage/src/client.rs

```diff
@@ -41,6 +43,9 @@ impl StorageClient {
         &self,
         certificate_id: CertificateId,
     ) -> Result<(PendingCertificateId, Certificate), StorageError> {
+        if self.sender.capacity() <= *constant::COMMAND_CHANNEL_CAPACITY {
```

Comment:
Here as well, would be good to read why and when this happens.

Reply:
Mmh I don't understand, when what happen?
The `if` is self-explanatory for me. It compare the current channel sender `capacity` to a constant, if it's true we just increase the metric counter.